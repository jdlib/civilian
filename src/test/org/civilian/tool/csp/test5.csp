package org.civilian.template.csp


template(String name) extends BaseTemplate(name, 5, "10")
{{
	<%name%>
}}