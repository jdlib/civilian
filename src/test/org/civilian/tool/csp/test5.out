/**
 * Generated from test5.csp
 * Do not edit.
 */
package org.civilian.template.csp;


import org.civilian.template.Template;


public class Test5 extends Template
{
	public Test5(String name)
	{
		super(name, 5, "10");                                           // line 6: super(name, 5, "10");
		this.name = name;
	}


	@Override protected void print() throws Exception
	{
		out.print(name);                                                // line 7: <%name%>
		out.printlnIfNotEmpty();
	}


	private String name;
}
